components/scheduler/views/agenda.md

```diff
@@ -35,10 +35,14 @@ The following parameters allow you to configure the Agenda view:
 >caption Declare the Agenda view in the markup
 
 ````CSHTML
-@* Define the Agenda view. *@
+@* Control the empty days visibility *@
```

Comment:
This comment seems unnecessary, because the variable names and UI labels on the next two lines speak for themselves.